netlogo-gui/src/main/app/InterfacePanel.scala

```diff
@@ -344,7 +329,16 @@ class InterfacePanel(val viewWidget: ViewWidgetInterface, workspace: GUIWorkspac
     }
   }
 
-  def keyPressed(evt: KeyEvent): Unit = { }
+  def keyPressed(evt: KeyEvent): Unit = {
```

Comment:
Why do you handle Ctrl+Z / Ctrl+Y yourself? why not deleting the `keyPressed` functionality and letting the Edit menu take care of undo/redo using its menu-items?